pkg/jobs/update.go

```diff
@@ -263,6 +266,45 @@ WHERE id = $1
 		}
 	}
 
+	v, err := u.txn.GetSystemSchemaVersion(ctx)
+	if err != nil {
+		return err
+	}
+	if v.AtLeast(clusterversion.V25_1_AddJobsTables.Version()) {
+		if ju.md.Status != "" && ju.md.Status != status {
+			if err := j.StatusStorage().Record(ctx, u.txn, "state", string(ju.md.Status)); err != nil {
```

Comment:
i'd think that if the job changes from running to paused, we'd want to update the job_status, even if progress was nil?